README.md

```diff
@@ -7,142 +9,94 @@
 [![codecov](https://codecov.io/gh/redis/jedis/branch/master/graph/badge.svg?token=pAstxAAjYo)](https://codecov.io/gh/redis/jedis)
 [![Discord](https://img.shields.io/discord/697882427875393627?style=flat-square)](https://discord.gg/qRhBuY8Z)
 
-# Jedis
-
-Jedis is a blazingly small and sane [Redis](http://github.com/antirez/redis "Redis") java client.
-
-Jedis was conceived to be EASY to use.
-
-Jedis is fully compatible with redis 2.8.x, 3.x.x and above*.
-
```

Comment:
`blazingly small and sane` is a brilliant description and doesn’t need to be modified.